sound/effect.h

```diff
@@ -427,6 +427,7 @@ class Effect {
         talkie.Say(talkie_font_directory_15, 15);
         talkie.Say(talkie_too_long_15, 15);
       } else if (strlen(dir)) {
+	STDOUT.println(" Font directory not found.");
```

Comment:
Use the new syntax, and maybe add the actual directory, like:

STDOUT << "Font directory not found: " << dir << "\n";

Reply:
Cool, done. Wasn't sure how else newline would be done if not using println. I did what works but is there a better way?
Also, how about this addition to exclude the "too long" message unless it's a Teensy?